frontend/src/lib/taxonomy.tsx

```diff
@@ -826,6 +826,14 @@ export function getKeyMapping(
                 description: `The response value for the ${index}${suffix} question in the survey.`,
             }
         }
+    } else if (value.startsWith('$survey_responded/')) {
```

Comment:
this one's already on line 800, just edit that?

Reply:
isn't that for the event property `survey_response`?

Reply:
oh no wait it has a slash in it. I assume `survey_response_` handles `survey_response` from the event property?

Reply:
Yeah I think I added it a while ago LOL nvm

Reply:
There's a separate one in KEY_MAPPINGS for the response. These ones are needed only when it's a programmatic property, i.e some part changes, like the id